src/trialsynth/base/models.py

```diff
@@ -0,0 +1,342 @@
+from typing import Union, Optional
+import logging
+
+from bioregistry import curie_to_str
+from indra.ontology.standardize import standardize_name_db_refs
+from indra.statements.agent import get_grounding
+
+logger = logging.getLogger(__name__)
+
+
+class SecondaryId:
+    """Secondary ID for a trial
+
+    Attributes
+    ----------
+    ns : str
+        The secondary ID's namespace
+    id : str
+        The ID of the secondary ID
+    """
+    def __init__(self, ns: str = '', id: str = ''):
```

Comment:
This looks odd to me, perhaps the code instantiating a SecondaryId should not make an instance of the SecondaryId is effectively going to contain placeholder empty strings